### ISSUE TITLE
Support for Terraform Beta installs

### ISSUE THREAD

**Issue by devorbitus** [warrensbox/terraform-switcher#39]
I understand you may want to keep the version list as clean as possible but it would be nice if you supported an override or something to be able to install the beta version of Terraform like https://releases.hashicorp.com/terraform/0.12.0-beta2/terraform_0.12.0-beta2_darwin_amd64.zip

**Comment by warrensbox** [comment]
I will look at the code tonight

**Comment by warrensbox** [comment]
This a useful feature, especially if a user wants to test out a terraform beta release version.
A couple of developers suggested that we pass a parameter to list all the version (including betas). The default, however, will only show the official release.


For example:-
The default `tfswitch` will display:
```
  ▸ 0.11.7
    0.9.8
    0.11.13
    0.11.12
↓   0.11.11
```
The optional arg  `tfswitch --list-all` or `tfswitch -l` will display all versions including rc and betas:
```
  ▸  0.12.0-rc1
     0.12.0-beta2
     0.12.0-beta1
     0.12.0
     0.11.13
↓    0.11.11
```

I will merge your changes to master and incorporate my changes to allow optional parameter `--list-all` list all tf version. 
I will release this new feature on Monday (May 20).

**Comment by warrensbox** [comment]
@devorbitus 
Would you be able test the changes before I release it?

Steps:
1. git checkout develop (your changes are merged to develop)
2. `go build -v -o tfswitch`
3. `./tfswitch 0.12.0-rc1` then `terraform -v` to verify 
4. `./tfswitch -l` or `./tfswitch --list-all` - verify if it list all the versions

**Comment by devorbitus** [comment]
@warrensbox Tested it as well as looked over the code and it looks good to me.

**Comment by devorbitus** [comment]
You may want to make some reference to the new feature on the [documentation page](https://warrensbox.github.io/terraform-switcher)

**Comment by warrensbox** [comment]
I have released this feature with #45 

It should install any version except for:
* 0.12.0-alpha4
* 0.12.0-alpha3

That's because the file was named incorrectly on the hashicorp release page.

For example, the file for `0.12.0-alpha2` looks like below:
<img width="469" alt="Screen Shot 2019-05-20 at 9 46 34 PM" src="https://user-images.githubusercontent.com/38867521/58374808-483f0180-7f0a-11e9-9c00-cf3de3cde143.png">


However the the file for `0.12.0-alpha4` looks like below:
<img width="668" alt="Screen Shot 2019-05-20 at 9 46 18 PM" src="https://user-images.githubusercontent.com/38867521/58374815-5d1b9500-7f0a-11e9-8b3a-7bf68a30cb59.png">

Notice the difference in how the file is named. I am assuming this might be an error on Hashicorp and I will open an issue with them. So far I have only noticed the discrepancy in these 2 versions - 0.12.0-alpha4 and 0.12.0-alpha3.
